rdrf/rdrf/views/patient_view.py

```diff
@@ -25,6 +25,11 @@
 from django.utils.translation import ugettext as _
 
 from rdrf.forms.dynamic.registry_specific_fields import RegistrySpecificFieldsHandler
+from rdrf.helpers.constants import (
```

Comment:
All these look like very patient form specific constants.
I would prefer keeping them local in this file instead of having them in a more general location like `rdrf.helpers.constants` and then importing them.
Any reason not to do so?

Reply:
I'm also using them in rdrf/admin_forms so I wanted to put them in a common location.